File: backend/src/main/java/com/example/stamp_app/repository/MeasuredDataMasterRepository.java
package com.example.stamp_app.repository;

import com.example.stamp_app.entity.MeasuredDataMaster;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;
import java.util.UUID;

public interface MeasuredDataMasterRepository extends JpaRepository<MeasuredDataMaster, UUID> {

    List<MeasuredDataMaster> findAll();

}

File: backend/src/main/java/com/example/stamp_app/repository/MeasuredDataMasterRepository.kt
package com.example.stamp_app.repository

import com.example.stamp_app.entity.MeasuredDataMaster
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface MeasuredDataMasterRepository : JpaRepository<MeasuredDataMaster?, UUID?> {

    override fun findAll(): List<MeasuredDataMaster>

}
